protos/schemas/common.fbs

```diff
@@ -48,14 +49,18 @@ table Witness {
     data:           [Bytes];
 }
 
+table CellDep {
+    tx_hash:        Bytes32;
+    index:          uint32;
+    is_dep_group:   uint8;
+}
+
 table OutPoint {
```

Comment:
`OutPoint` is useless after we changed deps struct, please remove it from `common.fbs` and `protocol.fbs`

Reply:
Oops, not realized that.

Reply:
We can't remove it because it used in syscall to load out point (used by contract)